File: Python algo/strings/1-reverse.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        for i in range(len(s)//2):
            tmp = s[i]
            s[i] = s[len(s)-1-i]
            s[len(s)-1-i] = tmp

        return s


sol = Solution()
print(sol.reverseString(['1','2','3','4','5','6']))

File: js algo/easy/strings/1-reverse.js
/**
 * @param {character[]} s
 * @return {void} Do not return anything, modify s in-place instead.
 */
 var reverseString = function(s) {
    
    for (let i = 0; i < parseInt(s.length/2); i++) {
        var tmp = s[i]
        s[i] = s[s.length-1-i]
        s[s.length-1-i] = tmp
    }
};

var x = ["h","e","l","l","o"]
reverseString(x)
console.log(x)